src/App.js

```diff
@@ -1,11 +1,43 @@
-import React from 'react';
-
-function App() {
-  return (
-    <div className="App">
-      <h1>FlashHub</h1>
-    </div>
-  );
-}
+import React, { Component } from 'react';
+import { Container } from 'semantic-ui-react';
+import Flashcard from './components/Flashcard';
+import axios from 'axios';
+
+class App extends Component {
+  state = {
+    flashcards: []
+  };
+
+  componentDidMount() {
+    axios.get('http://localhost:3000/api/flashcards')
+      .then(response => {
+        this.setState({
+          flashcards: response.data
+        })
+      })
+  };
+
+  render() {
+    const flashcards = this.state.flashcards
+    let flashcardDisplay
+
+    if ((flashcards.length = 1)) {
```

Comment:
can you try 
`if (flashcards.length >= 1) {`

Reply:
>= 1 also returns all articles...